### ISSUE TITLE
Request uses schema - stripping example values

### ISSUE THREAD

**Issue by IonGyth** [kminami/apib2swagger#29]
I'm attempting to convert a blueprint api file with example values for both request and response. The response works fine, given example values, it uses the values in the object body to provide examples for the api. The request, however, uses the schema which strips any example values.

example:
```FORMAT: 1A

# Blog Posts [/posts]

## Retrieve All Posts [GET]
+ Request (application/json)
    + Attributes (array[BlogPost], fixed-type)

## Data Structures

### BlogPost (object)
+ id: 42 (number, required)
+ text: Hello World (string)
+ author (Author) - Author of the blog post.

### Author (object)
+ name: Boba Fett
+ email: fett@intergalactic.com
```
converts to:
```swagger: '2.0'
info:
  title: ''
  version: ''
  description: ''
paths:
  /posts:
    get:
      responses: {}
      summary: Retrieve All Posts
      description: ''
      tags: []
      parameters:
        - name: body
          in: body
          schema:
            type: array
            items:
              type: object
              properties:
                id:
                  type: number
                text:
                  type: string
                author:
                  type: object
                  properties:
                    name:
                      type: string
                    email:
                      type: string
                  description: Author of the blog post.
              required:
                - id
definitions:
  Blog Posts: {}
  BlogPost:
    type: object
    required:
      - id
    properties:
      id:
        type: number
      text:
        type: string
      author:
        type: Author
  Author:
    type: object
    properties:
      name:
        type: string
      email:
        type: string
```
(notice no examples) when I really want:
```swagger: '2.0'
info:
  title: ''
  version: ''
  description: ''
paths:
  /posts:
    get:
      responses: {}
      summary: Retrieve All Posts
      description: ''
      tags: []
      parameters:
        - name: body
          in: body
          schema:
            type: array
            items:
              type: object
              properties:
                id:
                  type: number
                  example: 42
                text:
                  type: string
                  example: Hello World
                author:
                  type: object
                  properties:
                    name:
                      type: string
                      example: Boba Fett
                    email:
                      type: string
                      example: fett@intergalactic.com
                  description: Author of the blog post.
              required:
                - id
definitions:
  Blog Posts: {}
  BlogPost:
    type: object
    required:
      - id
    properties:
      id:
        type: number
      text:
        type: string
      author:
        type: Author
  Author:
    type: object
    properties:
      name:
        type: string
      email:
        type: string
```
(the example: \<example value\>)

My question is there a particular reason why the schema is used for the request? Could we add example values to the request?

Thanks!

**Comment by kminami** [comment]
Fixed at v1.1.0.
Thanks!

**Comment by kminami** [comment]
v1.1.1 released. Thnaks!
